projects/demo/src/modules/app/utils/file-type-guards.utils.ts

```diff
@@ -0,0 +1,13 @@
+import {EXAMPLE_MAIN_FILE_NAME} from '../../interfaces/front-end-example';
+
+export function isTS(fileName: string): boolean {
+    return fileName === EXAMPLE_MAIN_FILE_NAME.TS || fileName.endsWith('.ts');
+}
+
+export function isLess(fileName: string): boolean {
+    return fileName === EXAMPLE_MAIN_FILE_NAME.LESS || `${fileName}`.endsWith('.less');
+}
+
+export function isMainComponentFile(fileName: string): boolean {
```

Comment:
`isMainComponentFile` what do you think about `isRootAnyComponentFile` or another?

`isMain` -> because of this, I begin to doubt and think that this is a `main.ts` file

Reply:
i am not sure that it is better naming.

all files can be in the root directory but not not all of them are main.

Reply:
what do you think about `isPrimaryComponentFile`?

Reply:
done